src/ornament.ts

```diff
@@ -236,15 +236,15 @@ export class Ornament extends Modifier {
   /** Set the upper accidental for the ornament. */
   setUpperAccidental(accid: string): this {
     const scale = this.renderOptions.fontScale / 1.3;
-    this.accidentalUpper = new Glyph(Tables.accidentalCodes(accid).code, scale);
+    this.accidentalUpper = new Glyph(Tables.accidentalCodesOld(accid).code, scale);
```

Comment:
So I assume eventually, this will revert back to accidentalCodes(...) once everything has been ported?

Reply:
yes this is right. It was a way to keep the change smaller.